pkg/executor/api.go

```diff
@@ -84,18 +107,12 @@ func (executor *Executor) getServiceForFunctionApi(w http.ResponseWriter, r *htt
 // stale addresses are not returned to the router.
 // To make it optimal, plan is to add an eager cache invalidator function that watches for pod deletion events and
 // invalidates the cache entry if the pod address was cached.
-func (executor *Executor) getServiceForFunction(fn *fv1.Function) (string, error) {
+func (executor *Executor) getServiceForFunction(fn *fv1.Function, et executortype.ExecutorType) (string, error) {
```

Comment:
Do we really need to pass the executor type? I think we have it in the function itself right?

Reply:
Done